.github/workflows/pr.yml

```diff
@@ -69,6 +69,15 @@ jobs:
         sudo apt-get -y install valgrind
         LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${PWD}/build/Release/lib valgrind  --show-leak-kinds=definite,indirect,possible,reachable --track-origins=yes --num-callers=40 --error-exitcode=3 ./build/Release/bin/hello 5 || exit 1
```

Comment:
No valgrind in Windows.

Reply:
valgrind is left only to Ubuntu, others should run on all OS. 
The problem is with `dynamic_hello` in Windows, 
https://github.com/htm-community/htm.core/pull/856/checks?check_run_id=825923866#step:12:10 
What is the alternative to LD_PRELOAD for Win?

Reply:
Dynamic_hello on Windows is a problem because the library must be built as a DLL not a lib.
How about skipping that one for now.